src/Core/Scheduler/Cache.js

```diff
@@ -132,13 +132,24 @@ const Cache = {
      * @function
      *
      * @param {number} [time]
+     *
+     * @return {Object} Statistics about the flush: <code>entry_count</code>
+     * gives the number of entries before flushing, <code>hit</code> the number
+     * of flushed entries, and <code>miss</code> the number of entries still in
+     * cache.
      */
     flush: (time = Date.now()) => {
+        const entry_count = data.size;
+        let hit = 0;
+
         data.forEach((entry, key, map) => {
             if (entry.lifetime < time - entry.lastTimeUsed) {
                 map.delete(key);
+                hit++;
```

Comment:
You're using hit/miss in a rather unconventionnal way :)
Generally, in the context of a cache, `hit` is the number of times the requested value was in the cache. And `miss` is how many times the value wasn't. 

So `hit` and `miss` should be updated in `get` or `has`.

Actually: you could delete completely the `has` method since testing the result of `get` gives you the same result.
Then updating `miss` or `hit` would be solely in the `get` method.

And `flush` would reset them - by the way you could specify in the documentation that `flush` returns the number of hits/misses since last flush.

Reply:
Yeah sorry, updated !